functions/go/inflate-helm-chart/README.md

```diff
@@ -0,0 +1,238 @@
+`releaseName`  | Replaces RELEASE_NAME in the chart template output | test
+`namespace`    | Sets the target namespace for a release (`.Release.Namespace` in the template) | my-namespace
+`valuesInline` | Values to use instead of default values that accompany the chart |  global: <br> &emsp; enabled: false <br> tests: <br> &emsp; enabled: false  
+`valuesFile`   | valuesFile is a remote or local file path to a values file to use instead of the default values that accompanied the chart. The default values are in '{chartHome}/{name}/values.yaml', where `chartHome` and `name` are the parameters defined above. | tmp/values.yaml <br> <br> https://raw.githubusercontent.com/config-sync-examples/helm-components/main/cert-manager-values.yaml
+`valuesMerge`  | ValuesMerge specifies how to treat ValuesInline with respect to Values. Legal values: 'merge', 'override' (default), 'replace'. | replace
+`includeCRDs`  | Specifies if Helm should also generate CustomResourceDefinitions, defaults to 'false' | true
+
+The only required field is `name`.
+
+<!--mdtogo-->
+
+## Examples
+
+<!--mdtogo:Examples-->
+
+### Example with kpt
+
+To inflate a remote minecraft chart, you can run the following command: 
+
+```shell
+$ kpt fn eval --image gcr.io/kpt-fn/inflate-helm-chart:unstable --network -- \
+name=minecraft \
+repo=https://itzg.github.io/minecraft-server-charts \
+releaseName=test
+```
+
+The key-value pairs after the `--` will be converted to a 'functionConfig' of kind
+`ConfigMap`. The above command will add two files to your directory, which you can view:
+
+```shell
+$ kpt pkg tree
+├── [secret_test-minecraft.yaml]  Secret test-minecraft
+└── [service_test-minecraft.yaml]  Service test-minecraft
+```
+
+### Example with kustomize
+You can specify your `functionConfig` via the `generators` field in your `kustomization.yaml` file:
+
+The `functionConfig` can be of type `InflateHelmChart`:
+
+```yaml
+generators:
+- |-
+  apiVersion: v1
+  kind: InflateHelmChart
+  metadata:
+    name: demo
+    annotations:
+      config.kubernetes.io/function: |
+        container:
+          network: true
+          image: gcr.io/kpt-fn/inflate-helm-chart:unstable
+  helmCharts:
+  - name: minecraft
+    repo: https://itzg.github.io/minecraft-server-charts
+    version: 3.1.3
+    releaseName: test
+```
+
+You can equivalently use a `functionConfig` of type `ConfigMap`:
+
+```yaml
+generators:
+- |-
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: demo
+    annotations:
+      config.kubernetes.io/function: |
+        container:
+          network: true
+          image: gcr.io/kpt-fn/inflate-helm-chart:unstable
+  data:
+    name: minecraft
+    repo: https://itzg.github.io/minecraft-server-charts
+    version: 3.1.3
+    releaseName: test
+```
+
+For both of the above kustomizations, you can use kustomize v4 to inflate
+the helm charts with the following command:
```

Comment:
Can you also link to the kustomize reference doc if any?

Reply:
There is no documentation for this in kustomize

Reply:
Ack